api/egl.xml

```diff
@@ -832,7 +832,7 @@
         <enum value="0x333B" name="EGL_COLOR_COMPONENT_TYPE_FLOAT_EXT"/>
         <enum value="0x333C" name="EGL_DRM_MASTER_FD_EXT"/>
         <enum value="0x333D" name="EGL_OPENWF_DEVICE_EXT"/>
-            <unused start="0x333E"/>
+        <enum value="0x333E" name="EGL_GL_COLORSPACE_BT2020_HLG_EXT"/>
```

Comment:
Since this enum is reserved for James, he'll have to give an OK.

Reply:
Sorry, this enum is already in use by a somewhat provisional extension that we've been shipping for a while:

https://github.com/aritger/eglstreams-kms-example/blob/master/proposed-extensions/EGL_NV_output_drm_flip_event.txt

Reply:
There are a few unused in this block reserved for James, which does have colorspace related enums, if OK with James; or you will have to request a new block of enums from Jon. Just make a PR with egl.xml, and block at end - looks like x3540- is start of free enums

Reply:
Ack. Thank you James. It means they're not continuous, which is unfortunate, but fine. I'll grab a new one and update the PR.